src/test/java/com/mxf/rest/ArticleDAOTest.java

```diff
@@ -0,0 +1,58 @@
+package com.mxf.rest;
+
+import static org.junit.Assert.*;
+
+import java.sql.Date;
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.mxf.rest.dao.ArticleDAOService;
+import com.mxf.rest.entity.Article;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ComponentScan
+@EnableAutoConfiguration
+public class ArticleDAOTest {
```

Comment:
If it is integration tests then pls change name to `ArticleDAOServiceIntegrationTest`